script/deploy/safe/propose-to-safe.ts

```diff
@@ -70,34 +79,52 @@ const main = defineCommand({
       required: true,
     },
   },
+  /**
+   * Executes the propose-to-safe command
+   * @param args - Command arguments including network, rpcUrl, privateKey, to address, and calldata
+   */
   async run({ args }) {
     const chain = getViemChainForNetworkName(args.network)
 
-    const config: SafeApiKitConfig = {
-      chainId: BigInt(chain.id),
-      txServiceUrl: networks[args.network.toLowerCase()].safeApiUrl,
+    if (!process.env.MONGODB_URI) {
+      throw new Error('MONGODB_URI environment variable is required')
     }
 
-    const safeService = new SafeApiKit(config)
+    const mongoClient = new MongoClient(process.env.MONGODB_URI)
+    const db = mongoClient.db('SAFE')
+    const pendingTransactions = db.collection('pendingTransactions')
 
     const safeAddress = networks[args.network.toLowerCase()].safeAddress
 
     const rpcUrl = args.rpcUrl || chain.rpcUrls.default.http[0]
     const provider = new ethers.JsonRpcProvider(rpcUrl)
     const signer = new ethers.Wallet(args.privateKey, provider)
 
-    const ethAdapter = new EthersAdapter({
-      ethers,
-      signerOrProvider: signer,
-    })
+    let protocolKit: Safe
+    try {
+      protocolKit = await Safe.init({
+        provider: rpcUrl,
+        signer: args.privateKey,
+        safeAddress,
+      })
+    } catch (error) {
+      console.error('Failed to initialize Safe protocol kit:', error)
```

Comment:
sometimes we use `console` and other times we use `consola`
let's stick to one library for consistency.